### PR TITLE
Docs  cards multi paredown

### PR DESCRIPTION
**Pull request by RichKummer**
### Related Ticket(s)

#1125 

### Description

Paredown exercise for Card, Card in card, Card link, and Feature card.

### Changelog

**New**

- Overview, anatomy images for Cards page

**Changed**

- Edited content on Cards page.
- Card -> Cards
- Edited content on Cards page.
- Moved content from Card link, Card in card, and Feature card pages to Cards.

**Removed**

- Card link, Card in card, Feature card pages removed.
- yaml file updated to remove above pages.

### CODE REVIEW

**Comment by shixiedesign** [comment]
Hey @RichKummer much improved. The 1st image treatment is not completely there yet. Let's discuss on slack with more visuals. Here's a few suggestions on other stuff: 

### Feature card 

1. I don't think we need the last callout (6) on anatomy image here. Based on [functional spec](https://ibm.box.com/s/5pckvtokzjgtsarqebou7q4j3cfyx7ow), the prop that controls size is called `Size` not type. It should be a modifier. So I'd expect the structure of page to be more like: 

```
## Feature card
### Modifyers

#### Size

"Feature card has two sizes: medium and large. Medium is the default size, occupying 8 columns ..."
[Anatomy image of medium feature card]
"The large feature card occupies 12 columns and allows for the addition of an eyebrow as well as body copy."
[Anatomy image of large feature card]

### Resources for Feature card
...
```

2. I thought we discussed not to add "(required)" or "(optional)" in anatomy's caption text here. Let's keep this info in the Content guidance table, so we don't have to update it in 2 places. 
image

3. Seeing repeated titles in card again 😂 

image

File: src/pages/components/cards.mdx [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+| ----------------------------------------------------------------------- | ------------- | -------- | --------- | ------------------------------------------- | ----------------------- |
+| Heading                                                                 | Text          | Yes       | 1         | 40 / 55                                    |                         |
+| Copy                                                                    | Text          | No        | 1         | 65 / 85                                    | Hover over copy on bottom-aligned pictogram only. |
+| Pictogram                                                               | Element       | Yes       | 1         | –                                          | Top- or bottom-aligned |
+
+
+## Card in card
+
+Card in card shares the same functionality and text content requirements of a standard card, but requires a large media. Card in card is often used in a [Card group](https://www.ibm.com/standards/carbon/components/card-group) to as a featured card.
```
Comment (janchild) [inline src/pages/components/cards.mdx:149]:
```suggestion
Card in card shares the same functionality and text content requirements of a standard card, but requires a large media element. Card in card is often used in a [Card group](https://www.ibm.com/standards/carbon/components/card-group) as a featured card.
```

File: src/pages/components/cards.mdx [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+<Column colMd={8} colLg={8}>
+
+![Image of Card in card component](../../images/component/card/card-options-card-in-card-anatomy.png)
+
+</Column>
+</Row>
+
+1. **Media:** Can be an image or a video.
+2. **Heading:** Heading has the highest information hierarchy on a card, and should be succinct and descriptive.
+3. **Body copy:** Optional body copy can be added for additional descriptions.
```
Comment (janchild) [inline src/pages/components/cards.mdx:161]:
```suggestion
3. **Body copy:** Optional body copy can be added for additional description.
```

File: src/pages/components/cards.mdx [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+| Heading                                                                 | Text          | Yes      | 1         | 75 / 100                                    |                         |
+| Media                                                                   | [Image](https://www.ibm.com/standards/carbon/components/images/) or [Video](https://www.ibm.com/standards/carbon/components/video)          | Yes       | 1         | –                                     | Full width, 16:9 aspect ratio                        |
+| [CTA type](https://www.ibm.com/standards/carbon/components/cta)         | Option list   | Yes      | 1         | –                                           |                         |
+| [CTA text](https://www.ibm.com/standards/carbon/components/cta)         | Link          | No       | 1         | –                                           | URL assigned in setup   |
+
+## Card link
+
+Unlike Card, which can stand on its own, Card link should be presented with copy - it should never appear on its own. Card links contain a heading, an optional CTA (call-to-action), and optional body copy. The entire card is clickable.
```
Comment (janchild) [inline src/pages/components/cards.mdx:179]:
```suggestion
Unlike Card, which can be used as a standalone component, Card link should always be accompanied by contextual copy on the page. Card links contain a heading, an optional CTA (call-to-action), and optional body copy. The entire card is clickable.
```

File: src/pages/components/cards.mdx [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+<Row>
+<Column colMd={8} colLg={8}>
+
+![Image of Table of contents component](../../images/component/card/card-options-card-link-anatomy.png)
+
+</Column>
+</Row>
+
+1. **Heading:** Heading has the highest information hierarchy on a card, and should be succinct and descriptive.
+2. **Body copy:** Optional body copy can be added for additional descriptions.
```
Comment (janchild) [inline src/pages/components/cards.mdx:190]:
```suggestion
2. **Body copy:** Optional body copy can be added for additional description.
```

File: src/pages/components/cards.mdx [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+3. **CTA type:** When using an icon, by default will use `ArrowRight`.
```
Comment (janchild) [inline src/pages/components/cards.mdx:191]:
```suggestion
3. **CTA type:** When using an icon, the default is `ArrowRight`.
```

File: src/pages/components/cards.mdx [hunk 6]
```diff
@@ -0,0 +1,257 @@
...
+
+![Image of Card in card component](../../images/component/card/card-options-card-in-card-anatomy.png)
+
+</Column>
+</Row>
+
+1. **Media:** Can be an image or a video.
+2. **Heading:** Heading has the highest information hierarchy on a card, and should be succinct and descriptive.
+3. **Body copy:** Optional body copy can be added for additional descriptions.
+4. **CTA type:** When using an icon, by default will use `ArrowRight`.
```
Comment (janchild) [inline src/pages/components/cards.mdx:162]:
```suggestion
4. **CTA type:** When using an icon, the default is `ArrowRight`.
```

File: src/pages/components/cards.mdx [hunk 7]
```diff
@@ -0,0 +1,257 @@
...
+<Column colMd={8} colLg={8}>
+
+![Image of Feature card - medium](../../images/component/card/card-options-feature-card-medium-anatomy.png)
+
+</Column>
+</Row>
+
+1. **Image:** Must use a 1:1 aspect ratio.
+2. **Heading:** Heading has the highest information hierarchy on a card, and should be succinct and descriptive.
+3. **CTA type:** When using an icon, by default will use `ArrowRight`.
```
Comment (janchild) [inline src/pages/components/cards.mdx:222]:
```suggestion
3. **CTA type:** When using an icon, the default is `ArrowRight`.
```

File: src/pages/components/cards.mdx [hunk 8]
```diff
@@ -0,0 +1,257 @@
...
+4. **Type:** Determines the size of the Feature card.
+
+The large feature card occupies 12 columns and allows for the addition of an eyebrow as well as body copy.
```
Comment (janchild) [inline src/pages/components/cards.mdx:225]:
```suggestion
The large Feature card occupies 12 columns and allows for the addition of an eyebrow as well as body copy. 
```

File: src/pages/components/cards.mdx [hunk 9]
```diff
@@ -0,0 +1,257 @@
...
+![Image of Feature card - large](../../images/component/card/card-options-feature-card-large-anatomy.png)
+
+</Column>
+</Row>
+
+1. **Image:** Must use a 1:1 aspect ratio.
+2. **Eyebrow:** Add additional context to the the heading.
+3. **Heading:** Heading has the highest information hierarchy on a card, and should be succinct and descriptive.
+4. **Body copy:** Optional body copy can be added for additional descriptions.
+5. **CTA type:** When using an icon, by default will use `ArrowRight`.
```
Comment (janchild) [inline src/pages/components/cards.mdx:239]:
```suggestion
5. **CTA type:** When using an icon, the default is `ArrowRight`.
```

**Review by janchild — COMMENTED** [review]
Hi @RichKummer This is looking great! I went through with a fine-tooth comb making sure we were being super precise given the heavy lifting this page is doing. I'm happy to jump on a call if you want to talk through anything together. 🥂

**Comment by shixiedesign** [comment]
Also wanna pick on this word “type”. It’s been repeated used for CTA type (local, download, video etc). How about
> and the various ~types~ designs available work with a wide range of content.

image

**Review by janchild — COMMENTED** [review]
Hey @RichKummer, I still need to do the final read-through but wanted to ask if we could try something on the Overview image. I'm adding this as the example to the template and I think it would be worth trying a slightly different treatment on the labels. 

I can see that `mono` type helps to separate the label from the images themselves but maybe one size down would have it compete less with the images. Let me know if you want to discuss or hop on a call.

**Comment by RichKummer** [comment]
> Hey @RichKummer, I still need to do the final read-through but wanted to ask if we could try something on the Overview image. I'm adding this as the example to the template and I think it would be worth trying a slightly different treatment on the labels.
> 
> I can see that `mono` type helps to separate the label from the images themselves but maybe one size down would have it compete less with the images. Let me know if you want to discuss or hop on a call.

I think one size down is good. I tried two sizes down and felt the type was harder to read. Just pushed an update with the type size one less than the prev build

**Comment by shixiedesign** [comment]
Hey @RichKummer yeah I think Jan is right. I just took a look at the lastest, just wanna double check 1 thing. Is the letter spacing on the mono labels correct? Should be 0.32. 

image

**Comment by RichKummer** [comment]
> Hey @RichKummer yeah I think Jan is right. I just took a look at the lastest, just wanna double check 1 thing. Is the letter spacing on the mono labels correct? Should be 0.32. Felt a little close together.
> 
> image

It was a little bit off but pushing an update with the correct character spacing

File: src/pages/components/cards.mdx [hunk 10]
```diff
@@ -0,0 +1,257 @@
...
+
+While the entire default Card is clickable, the Static modifier makes it so only the text link is clickable on the card. This is useful for when you have multiple interactive elements on a single card such as CTAs and Tag links.
+
+<Row>
+
+<Column colMd={4} colLg={4}>
+
+![Example image: single selection hover state of a parent tier filter item.](../../images/component/card/cards-modifier-card-static-outline.png)
+
+<Caption>Example of a Card with the static modifier and an outline</Caption>
```
[the remark this answers is not in the document]
Comment (RichKummer) [reply src/pages/components/cards.mdx]: Agreed - I updated the language to hopefully be a little clearer. Also included a new gif animation to show how Card static can have multiple links to different destinations on one card.

**Review by shixiedesign — APPROVED** [review]
LGTM!

**Review by janchild — APPROVED** [review]
Woohoo!! This is great, @RichKummer!!! 🎉

**Pull request merged**